app/lib/common/providers/chat_providers.dart

```diff
@@ -34,3 +34,12 @@ final selectedChatIdProvider =
     NotifierProvider<SelectedChatIdNotifier, String?>(
   () => SelectedChatIdNotifier(),
 );
+
+final chatComposerDraftProvider = FutureProvider.autoDispose
```

Comment:
this is just moved from chat module to much more ```common``` providers section because it makes sense but also to avoid import conflicts.